content/events/2016-09-13-realitime-apps-with-node.js-and-rethinkdb.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Realtime Apps with Node.js and RethinkDB
+type: event
+date: 2016-09-13
+starttime: 18:00
```

Comment:
Beware that start/end times were different per event. Where did you get this from? The meetup page is probably the most reliable: https://www.meetup.com/NottsJS/events/233944192/

I guess this can be fixed retroactively - it doesn't matter for now.

Reply:
I was just using the `past.html` for data so set them all to the same, I've fixed them now. Also noticed there are a bunch of meetups that [were](https://www.meetup.com/en-AU/NottsJS/events/236607141/) [never](https://www.meetup.com/en-AU/NottsJS/events/240755008/) [listed](https://www.meetup.com/en-AU/NottsJS/events/243270192/) on https://nottsjs.org.